core/web/helpers.go

```diff
@@ -33,6 +33,29 @@ func jsonAPIError(c *gin.Context, statusCode int, err error) {
 	}
 }
 
+func paginatedResponseWithMeta(
+	c *gin.Context,
+	name string,
+	size int,
+	page int,
+	resource interface{},
+	count int,
+	err error,
+	meta map[string]string,
+) {
+	if errors.Cause(err) == orm.ErrorNotFound {
```

Comment:
This function has a bit of duplicated logic with the function below it `paginatedResponse`.  The only difference is that this function calls `NewPaginatedResponseWithMeta` whilst the `paginatedResponse` calls `NewPaginatedResponse`.  I'm open to suggestions on how this can be improved.